components/brave_new_tab_ui/api/initialData.ts

```diff
@@ -19,6 +20,7 @@ export type InitialData = {
   customImageBackgrounds: NewTab.ImageBackground[]
   braveRewardsSupported: boolean
   braveTalkSupported: boolean
+  braveVPNSupported: boolean
```

Comment:
Generally better to avoid adding stuff to initial data and drilling it down in favor of just accessing `loadTimeData` directly